components/scripts/lib/cli-parsers/maven/03-cli-parser.m4

```diff
@@ -24,6 +24,7 @@ function print_help() {
   print_option_usage "-1, --first-build-ci" "Specifies the URL for the build scan of the first build run by a CI agent."
   print_option_usage "-2, --second-build-ci" "Specifies the URL for the build scan of the second build run by a CI agent."
   print_option_usage -m
+  print_option_usage -x
```

Comment:
It seems here we need to explicitly define `ARG_OPTIONAL_BOOLEAN([fail-if-not-optimized],[x],[])` since this script is not pulling in the _common_ script.

Reply:
Fixed.

Reply:
Thanks! Good catch.